File: 240210/두 사람/two-person.js
const lines = require('fs').readFileSync(0).toString().trim().split('\n');
const [firstAge, firstSex] = lines[0].split(' ');
const [secondAge, secondSex] = lines[1].split(' ');

console.log(firstAge >= 19 && firstSex === 'M' || secondAge >= 19 && secondSex === 'M' ? 1 : 0);

File: 240423/두 사람/two-person.py
firstAge, firstSex = input().split()
secondAge, secondSex = input().split()
firstAge = int(firstAge)
secondAge = int(secondAge)

if (firstAge >= 19 and firstSex == 'M') or (secondAge >= 19 and secondSex == 'M'):
    print(1)
else:
    print(0)